server/db/models/cart.js

```diff
@@ -11,12 +11,24 @@ db.on('error', console.error.bind(console, 'mongodb connection error: '));
 var Schema = mongoose.Schema;
 
 var cartSchema = new Schema({
-	user: [{type: mongoose.Schema.Types.ObjectId, ref: 'User'}],
-    items: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Item' }],
+	user: {type: mongoose.Schema.Types.ObjectId, ref: 'User'},
```

Comment:
Is the indentation here weird? it's weird on github but i'm not sure if that's real or not.